load/README.md

```diff
@@ -2,12 +2,12 @@
 
 This package provides generic utilities for blockchain load testing. We break load generation down into the following components:
 
-- tx generator(s)
 - tx issuer(s)
-- tracker
+- tx listener(s)
+- tracker(s)
```

Comment:
Should tracker stay singular here and in the graph below? Seems we still keep this to a single tracker in the code

Reply:
Interesting point. I thought it might be useful to have a tracker per issuer, if we have N issuers issuing N different kind of transactions, to have metrics separated by issuer (=transaction kind). We can always share the same tracker across all "agents" (issuer+listener+tracker).

Reply:
The tracker is also currently used explicitly for load test wide metrics ie. total transactions confirmed across all of the agents. This means that if we use the same tracker, the orchestrator is expected to call a single function to get the sum across all agents as opposed to needing to explicitly sum all of the trackers together.

Could we simplify this by keeping it to a single tracker instance that is shared across all of them? Alternatively, how should we make sure not to create confusion in the future if we support both.

Reply:
Good point, we can have a single tracker for now, which could later be a global tracker and have a tracker per agent on top of this global tracker later. I don't think it would be that confusing to have, but let's not do it now. Changed back to a single global tracker per orchestrator!